### PR TITLE
Anti_Flicker

### PR DESCRIPTION
**Pull request by Ubaydulloh99**
Camera Settings Anti_Flicker

image

### CODE REVIEW

File: src/QmlControls/AntiFlicker.qml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    Column {
+        anchors.top: antiFlickerText.bottom
+        anchors.topMargin: 20
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: parent.bottom
+
+        spacing: 5
+
+        ItemDelegate {
```
Comment (thanhkaist) [inline src/QmlControls/AntiFlicker.qml:53]: What is the purpose of ItemDelegate?  Use Item only or Rectangle with color:"transparent"

File: src/QmlControls/AntiFlicker.qml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            Text {
+                text: 'Turn Off'
+                font.pixelSize: ScreenTools.mediumFontPointSize * 2.5
+                color: white
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.horizontalCenter: parent.horizontalCenter
+            }
+        }
+
+        ItemDelegate {
```
Comment (thanhkaist) [inline src/QmlControls/AntiFlicker.qml:66]: What is the purpose of ItemDelegate? Use Item only or Rectangle with color:"transparent"

File: src/QmlControls/AntiFlicker.qml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            Text {
+                color: white
+                text: qsTr("50Hz")
+                font.pixelSize: ScreenTools.mediumFontPointSize * 2.5
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.horizontalCenter: parent.horizontalCenter
+            }
+        }
+
+        ItemDelegate {
```
Comment (thanhkaist) [inline src/QmlControls/AntiFlicker.qml:79]: What is the purpose of ItemDelegate? Use Item only or Rectangle with color:"transparent"

File: src/QmlControls/AntiFlicker.qml (hunk 1 above)
Comment (Ubaydulloh99) [reply src/QmlControls/AntiFlicker.qml:53]: the reason is in Autel app also, itemdelegation has been used, like when you click it 

image

it has shadow and it give after effect shadow while usage of button

**Review by thanhkaist — APPROVED** [review]

**Pull request merged**